### PR TITLE
Fix description of SVG viewBox

### PR DESCRIPTION
**Pull request by rmartinho**
### Description

Changed "top left" to "bottom left" in the SVG viewBox description

### Motivation

The viewBox attribute specifies the *bottom* left coordinates, not the top left.

### CODE REVIEW

File: files/en-us/web/svg/attribute/viewbox/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec-urls: https://svgwg.org/svg2-draft/coords.html#ViewBoxAttribute
 
 The **`viewBox`** attribute defines the position and dimension, in user space, of an SVG viewport.
 
-The value of the `viewBox` attribute is a list of four numbers: `min-x`, `min-y`, `width` and `height`. The numbers `min-x` and `min-y` represent the top left coordinates of the viewport. The numbers `width` and `height` represent its dimensions. These numbers, which are separated by whitespace and/or a comma, specify a rectangle in user space which is mapped to the bounds of the viewport established for the associated SVG element (not the [browser viewport](/en-US/docs/Glossary/Viewport)).
+The value of the `viewBox` attribute is a list of four numbers: `min-x`, `min-y`, `width` and `height`. The numbers `min-x` and `min-y` represent the bottom left coordinates of the viewport. The numbers `width` and `height` represent its dimensions. These numbers, which are separated by whitespace and/or a comma, specify a rectangle in user space which is mapped to the bounds of the viewport established for the associated SVG element (not the [browser viewport](/en-US/docs/Glossary/Viewport)).
```
Comment (bsmth) [inline files/en-us/web/svg/attribute/viewbox/index.md:12]: This may be relative, depending on your perspective and how you conceptualize it, but the spec talks about min-x and min-y (minimum X and Y coordinates) for the position of the viewBox, and `(0, 0)` is top-left, so I would reword this whole thing which is mostly copied from the spec anyway:

```suggestion
The value of the `viewBox` attribute is a list of four numbers separated by whitespace and/or a comma: `min-x`, `min-y`, `width`, and `height`. `min-x` and `min-y` represent the smallest X and Y coordinates that the `viewBox` may have (the origin coordinates of the `viewBox`) and the `width` and `height` specify the `viewBox` size. The resulting `viewBox` is a rectangle in user space mapped to the bounds of the viewport of an SVG element (not the [browser viewport](/en-US/docs/Glossary/Viewport)).
When an SVG contains a `viewBox` attribute (often in combination with a [`preserveAspectRatio`](/en-US/docs/Web/SVG/Attribute/preserveAspectRatio) attribute), a transform stretches or resizes the SVG viewport to fit a particular container element.
```

I think there's room for improvement on this area in general.

See https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Positions#what_are_pixels

**Comment by bsmth** [comment]
Thanks a lot for your submission! I've left a comment above about SVG coords because I think this section needs a bit more information. There's a good diagram here which describes what's happening with the `viewBox`: https://docs.aspose.com/svg/net/drawing-basics/coordinate-systems-and-units/

Let me know what you think!

**Comment by rmartinho** [comment]
@bsmth sure, go ahead :)

**Review by bsmth — APPROVED** [review]
Thanks a lot @rmartinho, going to merge shortly!
